config/prometheus/prometheus-roles.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: prometheus
+rules:
+- apiGroups: [""]
+  resources:
+  - nodes
+  - nodes/proxy
+  - services
+  - endpoints
+  - pods
```

Comment:
If not too hard, maybe we can restrict the permission to the resources that are needed?

Reply:
Good idea.

Reply:
I still think this is a good idea, however much of the docs that I'm reading seem to suggest the same setup as is. I think I'll leave it for now unless we can pinpoint exactly what should be changed.

https://github.com/prometheus/prometheus/blob/main/documentation/examples/rbac-setup.yml
https://github.com/bibinwilson/kubernetes-prometheus/blob/master/clusterRole.yaml